### PR TITLE
Fix (#121) - Added type annotations

### PR DESCRIPTION
**Pull request by i-karen**
Please read https://github.com/terminusdb/terminus-client-python/blob/master/Contributing.md
before submitting a pull request.

Fix for #121 
https://github.com/terminusdb/terminusdb-client-python/issues/121

Added parameter and class anotations for **woql_view.py**.
The types of some parameters were ambigous - so I did not annotate them (example : input_radius, input_size, input_choice, input_obj etc ...)

### CODE REVIEW

**Review by Cheukting — CHANGES_REQUESTED** [review]
I have commented on the types that you are not sure, would you like to add them in?

File: terminusdb_client/woqlview/woql_view.py [hunk 1]
```diff
@@ -92,42 +93,42 @@ def color(self, input_color):
...
 
-    def size(self, input_size):
+    def size(self, input_size) -> WOQLView:
         if self.obj is None:
             raise SyntaxError("size() should be used following a node() or edge()")
         self.config += self.obj + f".size({input_size});\n"
         return self
 
-    def collision_radius(self, input_radius):
+    def collision_radius(self, input_radius) -> WOQLView:
```
[the remark this answers is not in the document]
Comment (i-karen) [reply terminusdb_client/woqlview/woql_view.py:110]: Maybe I can take a generic numeric type for these cases!

**Comment by i-karen** [comment]
Added the above changes. Please review again!

**Review by Cheukting — APPROVED** [review]
Looks good, thanks!

**Comment by Cheukting** [comment]
One last thing, since "from __future__ import annotations" is not supported on Python 3.6. I think you can skip it or only import it if the Python version is >= 3.7 (using sys.version)

**Comment by i-karen** [comment]
Okay, I can add a check there.
However, for less than Python 3.7, the class type annotation should be enclosed in strings as indicated here https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class

Any suggestions here?

**Comment by i-karen** [comment]
I added the selective import based on version?

Is there any import guideline from PEP or is this okay?
@Cheukting

**Comment by Cheukting** [comment]
@i-karen yea I can see the checker is complaining about that. Let me have a look and maybe we can switch that off.

File: terminusdb_client/woqlview/woql_view.py [hunk 2]
```diff
@@ -1,3 +1,9 @@
+import sys
+if sys.version_info[0]==3 and sys.version_info[1]>7:
+    from __future__ import annotations
```
Comment (Cheukting) [inline terminusdb_client/woqlview/woql_view.py:3]: for now, let's add "# noqa" at the end of this like to by-pass the flaks8 check. See if Travis pass this time.

**Review by Cheukting — CHANGES_REQUESTED** [review]
Almost there, thanks!

**Comment by i-karen** [comment]
Hey @Cheukting , I removed the annotations for class self as it was making things complicated.
Now, all the Travis CI builds pass!

**Comment by Cheukting** [comment]
@i-karen Cool, nice work! Thank you so much.

**Pull request merged**